README.md

```diff
@@ -71,27 +71,23 @@ AgentKit is organized as a [monorepo](https://en.wikipedia.org/wiki/Monorepo) th
 ### cdp-agentkit-core
 
 Core primitives and framework-agnostic tools that are meant to be composable and used via AgentKit framework extensions (ie, `cdp-langchain`).
-See [CDP Agentkit Core](./cdp-agentkit-core/README.md) to get started!
```

Comment:
Removing these since these READMEs don't exist, and we will be changing up the packages in the 0.1.0 release anyway.